lib/events/s3sessions/s3handler_config_test.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2022 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package s3sessions
+
+import (
+	"net/url"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/gravitational/teleport/lib/utils"
+)
+
+func TestMain(m *testing.M) {
+	utils.InitLoggerForTests()
+	os.Exit(m.Run())
+}
+
+func TestConfig_SetFromURL(t *testing.T) {
+
+	cases := []struct {
+		name         string
+		url          string
+		cfgAssertion func(*testing.T, *Config)
+	}{
+		{
+			name: "fips enabled via url",
+			url:  "s3://bucket/audit?insecure=true&disablesse=true&acl=private&use_fips_endpoint=true&sse_kms_key=abcdefg",
+			cfgAssertion: func(t *testing.T, config *Config) {
+
+				var (
+					expectedBucket = "bucket"
+					expectedACL    = "private"
+					expectedRegion = "us-east-1"
+					sseKMSKey      = "abcdefg"
+				)
+				require.EqualValues(t, expectedBucket, config.Bucket)
+				require.EqualValues(t, expectedACL, config.ACL)
+				require.EqualValues(t, sseKMSKey, config.SSEKMSKey)
+				require.EqualValues(t, expectedRegion, config.Region)
```

Comment:
`require.Equal` is a stricter check than `require.EqualValues`.

I would prefer the stricter one.